### PR TITLE
Fix invalid time in member contribution page

### PR DESCRIPTION
**Pull request by anishpawaskar**
**Closes:** https://github.com/Real-Dev-Squad/members-site/issues/99

**What is change?**
- if `startsOn` or `endsOn` key is false i.e undefined then completion date should show as 'N/A', so updated the `isStatusVerifiedOrNotVerified`  function.
- Also estimated date is not showing up in active task section so i have tweak title variable in `<ContributionAccordion />` based on the accordion title.
- Also write test for `ContributionAccordianItem` component.
- Fix the alignment of user details on smaller screen size devices. 
- Fix invalid data showing up in skills modal of member profile page.

Backend changes:
- [ ] Yes
- [x] No

Frontend Changes:
- [x] Yes
- [ ] No

Database changes:
- [ ] Yes
- [x] No

Breaking changes (If your feature is breaking/missing something please mention pending tickets)
- [ ] Yes
- [x] No

**Before:** 

https://github.com/Real-Dev-Squad/members-site/assets/22213872/1c716c5b-9d3c-4a85-9f58-5e245cfdbbfa

- Fix invalid data showing up in skills modal of member profile
Real Dev Squad - Google Chrome 06-11-2023 15_44_25

**After:**

https://github.com/Real-Dev-Squad/members-site/assets/22213872/31c4ddf7-b20c-4a23-a57a-268fb915ac70

- Fix invalid data showing up in skills modal of member profile

Real Dev Squad - Google Chrome 06-11-2023 15_46_03

**Test coverage:**
image
image

### CODE REVIEW

**Comment by ajoykumardas12** [comment]
Please add more PR details.

**Review by iamitprakash — CHANGES_REQUESTED** [review]
Please add required PR details

File: src/components/MemberContribution/ContributionAccordian.tsx [hunk 1]
```diff
@@ -40,14 +40,17 @@ export default function ContributionAccordion({
   const renderData = contribution?.map((data: any, idx: number) => {
     const task = setData(data);
     // title exist boolean
-    const title = !!data?.task?.title;
+    const isTitle =
```
Comment (ajoykumardas12) [inline src/components/MemberContribution/ContributionAccordian.tsx:43]: Does this variable track if something is title or other element(eg. paragraph)? If not then naming can be improved.

File: src/components/MemberContribution/ContributionAccordianItem.tsx [hunk 2]
```diff
@@ -7,15 +7,15 @@ import NextLink from "next/link";
 import { Button, Link } from "@chakra-ui/react";
 import styles from "./memberContribution.module.css";
 import Image from "next/image";
-import { useGetIsSuperUser } from "@/src/utils/customHooks";
+import { useGetIsSuperUser } from "../../utils/customHooks";
```
Comment (vinayak-trivedi) [inline src/components/MemberContribution/ContributionAccordianItem.tsx:10]: why is this changed?

File: src/components/MemberContribution/DeliveryDetails.tsx [hunk 3]
```diff
@@ -19,44 +18,64 @@ export default function DeliveryDetails(props: any) {
...
       timeStampComponent = (
         <>
           <Box>
-            <span>{ESTIMATED_COMPLETION}</span>
+            <span>{COMPLETED_IN}</span>
             <b>{data?.completionDuration}</b>
           </Box>
+          {data?.displayFeatureLiveDate && (
+            <Box>
+              <span style={{ color: "#90a4ae" }}>
```
Comment (vinayak-trivedi) [inline src/components/MemberContribution/DeliveryDetails.tsx:36]: can we put this styling in a CSS file?

File: src/components/MemberContribution/DeliveryDetails.tsx [hunk 4]
```diff
@@ -19,44 +18,64 @@ export default function DeliveryDetails(props: any) {
...
-            <b>{data?.completionDuration}</b>
-          </Box>
-          <Box>
-            <span style={{ color: '#90a4ae' }}>
-              {FEATURE_LIVE_ON} {data?.displayFeatureLiveDate}
-            </span>
-          </Box>
+          {isTaskCompletedOrDone ? (
+            <Box>
+              <span style={{ color: "#90a4ae" }}>{COMPLETED_IN}</span>
```
Comment (vinayak-trivedi) [inline src/components/MemberContribution/DeliveryDetails.tsx:53]: same as above, can we put this in CSS file?

**Review by vinayak-trivedi — CHANGES_REQUESTED** [review]
Great work @anishpawaskar , adding few comments, also there are a lot of changes where single quotes changed to double quotes, can we remove them as well?

File: src/components/MemberContribution/ContributionAccordian.tsx (hunk 1 above)
Comment (anishpawaskar) [reply src/components/MemberContribution/ContributionAccordian.tsx:43]: Yes it track the title of the task that's why i have name it isTitle

File: src/components/MemberContribution/ContributionAccordianItem.tsx (hunk 2 above)
Comment (anishpawaskar) [reply src/components/MemberContribution/ContributionAccordianItem.tsx:10]: As of now jest is not configured correctly in member site even for previous test i have change import

File: src/test/unit/components/member-contribution/ContributionAccordianItem.test.tsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    expect(container).toBeTruthy();
+    expect(isOptionKeyPressed).toBe(true);
+    fireEvent.mouseLeave(contributionContainer);
+
+    const { container: deliveryDetails } = renderWithProviders(
+      <DeliveryDetails isTitle={false} task={task} />
+    );
+    expect(deliveryDetails).toBeInTheDocument();
+
+    const contributionHeading = screen.getByRole("heading", { level: 3 });
```
[the remark this answers is not in the document]
Comment (anishpawaskar) [reply src/test/unit/components/member-contribution/ContributionAccordianItem.test.tsx]: Sure will try to get h3 element with a specific value.

**Review by ajoykumardas12 — APPROVED** [review]
